Sources/BraintreeAmericanExpress/BTAmericanExpressClient.swift

```diff
@@ -30,7 +30,10 @@ import BraintreeCore
         apiClient.sendAnalyticsEvent(BTAmericanExpressAnalytics.started)
 
         apiClient.get("v1/payment_methods/amex_rewards_balance", parameters: parameters) { [weak self] body, response, error in
-            guard let self = self else { return }
+            guard let self else {
+                completion(nil, BTAmericanExpressError.deallocated)
```

Comment:
We can't send analytics for these because using `apiClient` would rely on using `self`